helm/grafana/Chart.yaml

```diff
@@ -1,27 +1,20 @@
 apiVersion: v2
 name: grafana
-version: 1.1.0
+version: [[ .Version ]]
```

Comment:
```suggestion
version: 2.0.0
```

Reply:
Current CI fails if we have the wrong `version`, and suggests we set it to `"[[ .Version ]]`.

Reply:
If we use architect yes, with ABS we need to set the version manually

Reply:
I updated the Chart.yaml with this change